src/main/java/teammates/logic/core/EmailGenerator.java

```diff
@@ -171,6 +171,22 @@ private EmailWrapper generateFeedbackSessionEmailBaseForInstructorReminders(
     }
     
     /**
+     * Generates the feedback session closed emails for the given {@code session}.
+     */
+    public List<EmailWrapper> generateFeedbackSessionClosedEmails(FeedbackSessionAttributes session) {
+
+        boolean isEmailNeeded = fsLogic.isFeedbackSessionForStudentsToAnswer(session);
```

Comment:
That's different, because the emails are sent to the students and instructors get a copy. Here, the emails are intended for the instructors.